app/controllers/pages_controller.rb

```diff
@@ -4,4 +4,8 @@ class PagesController < ApplicationController
   def show
     render template: "pages/#{params[:page].tr('-', '_')}"
   end
+
+  def induction_tutor_materials
+    render template: "pages/induction_tutor_materials/#{params[:provider].tr('-', '_').downcase}/#{params[:year].tr('-', '_')}"
```

Comment:
Does the year param need replacing `-` with `_`? Also, there are some helper methods out there to make it neater to read, `underscore`?

Reply:
The downcase can be removed, just spotted that.

Not if we set the param as "year_one... i guess it's how we want the url to read. Either /year_one or /year-one? or another way.

At the minute year-one and year_one work. It's on the page itself it changes. If you type in the url /year_one but the param is year-one it won't have the Year 1 tab highlighted.

Reply:
Oh, my bad, I thought year was 2021, 2022, but it is `year_one` `year_two`. In that case let's keep them.